docs/install.md

```diff
@@ -66,11 +66,11 @@ can be insalled via the system package manager.
 #### Compile
 
 ```sh
-cmake -S . -B build -G Ninja
-cmake --build build/
+cmake --preset linux-release
+cmake --build --preset linux-release
 
 # Optionally, if you want to install from source
-cmake --build build/ --target install
+cmake --build --preset linux-release --target install
 ```
```

Comment:
this assumes Linux OS.

Maybe we could come up with a `default` target that is `linux-release` on linux, `macos-release` on mac, etc. I don't have a perfect answer to this one yet either (doesn't have to be addressed in this PR). 🤔